pyeapi/eapilib.py

```diff
@@ -259,11 +259,11 @@ def connect(self):
             self._tunnel()
         # If there's no CA File, don't force Server Certificate Check
         if self.ca_file:
-            self.sock = ssl.wrap_socket(sock, self.key_file, self.cert_file,
+            self.sock = ssl.SSLContext.wrap_socket(sock, self.key_file, self.cert_file,
```

Comment:
good catch, however it now breaks the formatting rules (the line extends past allowed column size),  needs some adjustment